changelog/README.md

```diff
@@ -0,0 +1,15 @@
+The `changelog` directory contains the description of the changes introduced into the repository.
+The changes are essentially divided into 4 categories:
+- changes: PRs bringing Changes and/or Enhancements
+- bugfixes: PRs fixing existing issues
+- security: PRs fixing security issues
+- updates: PRs updating packages
+
+## How to add the file
+
+Based on the category the PR falls into create a new file in the respective directory
+with the filename format `DD-MM-YYYY-<few-words-about-the-change>.md (can be generated via: `$(date '+%d-%m-%Y')-<few-words-about-the-change>.md``
+
+The contents of the file should describe the changes in an elaborative manner, except for `Updates`.
```

Comment:
```suggestion
The contents of the file should describe the changes in an elaborative manner (use the past tense for the change/bugfix description to avoid confusion with the imperative voice for actions the user should do as a result).
Security fixes of upstream packages and package updates can be kept short in most cases and follow a standard format.
```